### PR TITLE
chore(release): bump version to 0.13.31

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,43 @@
...
+
+### Features
+
+* allow custom args in `kubectl apply` for kubernetes deployments ([#6107](https://github.com/garden-io/garden/issues/6107)) ([85ab8e263](https://github.com/garden-io/garden/commit/85ab8e263))
+* **images:** add multi-platform images for garden deployed services ([#6072](https://github.com/garden-io/garden/issues/6072)) ([76fa956d0](https://github.com/garden-io/garden/commit/76fa956d0))
+* **template:** add date template helper functions ([#5997](https://github.com/garden-io/garden/issues/5997)) ([39d239661](https://github.com/garden-io/garden/commit/39d239661))
+
+### Improvements
+
+* improve cli startup performance using v8 cache ([#6049](https://github.com/garden-io/garden/issues/6049)) ([7d8034b3b](https://github.com/garden-io/garden/commit/7d8034b3b))
```
Comment (stefreak) [inline CHANGELOG.md:32]: Also a performance improvement technically

**Review by stefreak — DISMISSED** [review]
🚀

File: CHANGELOG.md (hunk 1 above)
Comment (vvagaytsev) [reply CHANGELOG.md:32]: Thanks! I'll fix the changelog and the release notes

**Review by stefreak — APPROVED** [review]
👍

**Pull request merged**